array/DNAm/analysis/EWAS/dmrff.r

```diff
@@ -0,0 +1,83 @@
+# ---------------------------------------------------------------------#
+##
+## Title: Regional analysis for case control within each cell type
+##
+## Purpose of script: identify regions of differential  DNA methylation for
+## schizophrenia vs controls for each cell type separately using dmrff
+##
+## ---------------------------------------------------------------------#
+
+library(dmrff)
+
+args <- commandArgs(trailingOnly = TRUE)
+dataDir <- args[1]
+cellType <- args[2]
+refPath <- args[3]
+
+normData<-file.path(dataDir, "3_normalised/normalised.rdata")
+resPath <- file.path(dataDir, "4_analysis/EWAS")
+
+load(normData)
+
+print(paste0("running dmrff on ", cellType, " cell type..."))
+## subset to cell type samples
+QCmetrics<-QCmetrics[which(QCmetrics$Cell.type == cellType),]
+
+# filter to schizophrenia and control only
+QCmetrics<-QCmetrics[QCmetrics$Phenotype %in% c("Schizophrenia", "Control"),]
```

Comment:
another hard coded ref to schizophrenia here

Reply:
See above comment.